applications/nrf5340_audio/doc/building.rst

```diff
@@ -223,6 +221,36 @@ Complete the following steps to build the application:
    Alternatively, you can add the ``-d`` parameter to the ``west`` command to specify a custom build folder. This lets you build firmware for both
    headset and gateway before programming any development kits.
 
+Application Configuration Files
+===============================
+
+The application uses a :file:`prj.conf` configuration file located in the sample root directory for the default configuration.
+It also provides additional files for different custom configurations.
+When you build the sample, you can select one of these configurations using the :makevar:`FILE_SUFFIX` variable.
+
+See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.
+
+The application supports the following configuration types:
+
+.. list-table:: Application Build Types
+   :widths: auto
+   :header-rows: 1
+
+   * - Build type
+     - File name
+     - Description
+   * - Debug (default)
+     - :file:`prj.conf`
+     - Debug version of the application. Provides full logging capabilities and debug optimizations to ease development.
+   * - Release
+     - :file:`prj_release.conf`
+     - Release version of the application. Disables logging capabilities and disables development features to create a smaller application binary.
+   * - FOTA DFU
+     - :file:`prj_fota.conf`
+     - | Builds the debug version of the application with the features needed to perform DFU over BLE, and includes bootloaders so that the applications on both the application core and network core can be updated.
```

Comment:
```suggestion
     - | Builds the debug version of the application with the features needed to perform DFU over Bluetooth LE, and includes bootloaders so that the applications on both the application core and network core can be updated.
```